### DISCUSSION TITLE
Working with Slivers

### DISCUSSION THREAD

**Discussion by diegogarciar** [flyerhq/flutter_chat_ui#645] [Ideas & Requests]
Hey, I'm trying to build a post comments page for my app, I am trying to mimic X/Threads app. I wanted to use flutter_chat_ui, but it's giving me a hard type synchronizing the scrollviews, has anyone done this or knows how to do it? I thought that by putting the same scrollController on both the Chat widget and the customScrollController, the scrolling would be synchronised

https://github.com/user-attachments/assets/fdf43e79-c50d-4eb0-bd5b-ac39dc48960a

**Comment by demchenkoalex** [comment]
hey :) i don't know much about how flutter work under the hood, but logically the simplest thing to do would be to add the post to the top of the chat scroll view? that is if you want post to move with the messages?
